test/e2e/flowaggregator_test.go

```diff
@@ -579,7 +594,25 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	}
 	// Checking only data records as data records cannot be decoded without template
 	// record.
-	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records: ", len(recordSlices))
+	assert.GreaterOrEqualf(t, dataRecordsCount, expectedNumDataRecords, "IPFIX collector should receive expected number of flow records. Considered records:", recordSlices)
+	// This is for debugging the intermittent failure.
+	if dataRecordsCount == 1 {
+		data.forAllMatchingPodsInNamespace("component=antrea-agent", antreaNamespace, func(nodeName, podName, nsName string) error {
+			cmd := fmt.Sprintf("kubectl -n %s logs -c antrea-agent %s", nsName, podName)
+			_, antreaLogs, _, err := provider.RunCommandOnNode(controlPlaneNodeName(), cmd)
+			t.Logf("Logs for %s: %v err: %v", podName, antreaLogs, err)
+			return err
+		})
```

Comment:
When the test fails, we can view the antrea-agent logs from GitHub workflow. Is this log for debugging Jenkins issue?

Reply:
This is a temporary addition for this PR to see if there are any intermittent kind CI failures and debug them. Will remove this before merging.